app/controllers/admin/members_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Admin
+  class MembersController < BaseController
```

Comment:
I see that we don't have it in the issue, but I think will be great to add `new` & `create` action. Let's add it?

Reply:
No need need this right now